protocol/receiveaddresses/receiveaddresses.go

```diff
@@ -0,0 +1,65 @@
+package receiveaddresses
+
+import (
+	"github.com/kaspanet/kaspad/addrmgr"
+	"github.com/kaspanet/kaspad/config"
+	"github.com/kaspanet/kaspad/netadapter/router"
+	peerpkg "github.com/kaspanet/kaspad/protocol/peer"
+	"github.com/kaspanet/kaspad/protocol/protocolerrors"
+	"github.com/kaspanet/kaspad/wire"
+	"time"
+)
+
+const timeout = 30 * time.Second
+
+// ReceiveAddresses asks a peer for more addresses if needed.
+func ReceiveAddresses(incomingRoute *router.Route, outgoingRoute *router.Route,
+	peer *peerpkg.Peer, addressManager *addrmgr.AddrManager) (routeClosed bool, err error) {
+
+	subnetworkID, err := peer.SubnetworkID()
+	if err != nil {
+		panic(err)
+	}
+
+	msgGetAddr := wire.NewMsgGetAddr(false, subnetworkID)
```

Comment:
Please rename everything related to GetAddr to GetAddresses and everything related to Addr to Addresses.